qemu/tests/qmp_command.py

```diff
@@ -19,167 +21,127 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environmen.
     """
-    def check_result(qmp_o, output=None, exception_list=""):
+
+    def check_list(qmp_o, key, val=None, check_item_in_pair=True):
+        """
+        Check if the expect key, val are contained in QMP output qmp_o.
+
+        :param qmp_o: output of QMP command
+        :type qmp_o: list
+        :param key: expect result
+        :type key: str
+        :param val: expect result
+        :type val: str or None(if check_item_in_pair=False)
+        :param check_item_in_pair: If expect result is dict or str
+        :type check_item_in_pair: bool. Means expect result is dict or str.
```

Comment:
True means dict and false means str? Had better make it clear.

Reply:
This param means check a dict (item in pair) or a str (can be key or value of the dict).
I think it is clear, any good suggestion for it ?

Reply:
@sitoliu  No, it should be True.

Reply:
I mean it's beter to explicitly tell what true and false means in the doc string.

Reply:
updated